include/clap/host.h

```diff
@@ -35,6 +35,12 @@ typedef struct clap_host {
    void(CLAP_ABI *request_process)(const struct clap_host *host);
 
    // Request the host to schedule a call to plugin->on_main_thread(plugin) on the main thread.
+   // This callback should be called as soon as practicable, usually in the host application's next
+   // available main thread time slice. Typically callbacks occur at least in the 10s-to-50s-of-milliseconds
```

Comment:
maybe:
Typically callbacks occur within at 33ms (~30Hz).

Maybe just giving an idea of the worst case is enough.